### PR TITLE
docs for PLA-94

### PR DESCRIPTION
**Pull request by paulosimao-ardanlabs**

### CODE REVIEW

File: .doc/apps/di-braze-connector/Readme.md [hunk 1]
```diff
@@ -1,14 +1,136 @@
...
+            - name: GOOGLE_APPLICATION_CREDENTIALS
+              value: /gcp-key/key
+            - name: PROJECT_ID
+              value: uw-data-platform-dev
+            - name: WAIT_SIG
+              value: 'false'
+            - name: "KAFKA_BROKERS"
+              value: "kafka:9092"
+            - name: "KAFKA_GROUP_ID"
+              value: "di-braze-connector"
```
Comment (AnthonyArdronUW) [inline .doc/apps/di-braze-connector/Readme.md:93]: Could this be `*app` instead ?

Comment (paulosimao-ardanlabs) [reply .doc/apps/di-braze-connector/Readme.md:93]: sure

File: .doc/apps/di-braze-connector/Readme.md [hunk 2]
```diff
@@ -1,14 +1,136 @@
 # di-braze-connector
 The Braze connector works in a similar way compared to BigQuery Connector.
 
-At every message received in GCS upon a new file being saved, this connector will evaluate and depending on the file (if is related to a DD with egress in Braze),
-will trigger the worflow as shown below.
+At every message received by the API, a file is saved into GCS.
+Once the batch is over, the api will trigger braze using kafka.
```
Comment (catalin-ilea) [inline .doc/apps/di-braze-connector/Readme.md:5]: typo `once`

File: zarf/manifests/di-braze-connector/app.yaml [hunk 3]
```diff
@@ -70,13 +68,11 @@ spec:
...
               value: "di-platform-api:50000"
           volumeMounts:
-            - mountPath: /defs
-              name: data-products-definitions
             - mountPath: /gcp-key
               name: gcp-key
             - mountPath: /badger
               name: data-infra-braze-connector-pvc
-          image: registry.uw.systems/data-platform/di-braze-connector:latest
+          image: registry.uw.systems/data-platform/di-braze-connector:cd7e8ffd2da698165a377f1b4593e8e243ced07e
```
Comment (catalin-ilea) [inline zarf/manifests/di-braze-connector/app.yaml:75]: Why are we pinning this in the base?

Comment (paulosimao-ardanlabs) [reply zarf/manifests/di-braze-connector/app.yaml:75]: I understood this should be ok - given version issues we discussed with new products. Should we always point to latest here, and leave versioning to the deployments?

**Review by catalin-ilea — APPROVED** [review]

**Review by catalin-ilea — APPROVED** [review]

**Pull request merged**